site/docs/.vitepress/config.ts

```diff
@@ -50,5 +51,21 @@ export default defineConfig({
     build: {
       chunkSizeWarningLimit: 1600,
     },
+    resolve: {
+      alias: [
+        {
+          find: /^.*\/VPNavBar\.vue$/,
+          replacement: fileURLToPath(
+            new URL("./components/CustomNavBar.vue", import.meta.url),
```

Comment:
Isn't there `import.meta.dirname`?

Reply:
That works as well. I've set a minimum required version of Node.js as a safeguard, considering it's a new API. (https://github.com/grammyjs/website/pull/1011/commits/ab639136e59193762dbfb699718e8f54ceb89e0c)